ansible-scylla-node/tasks/Debian.yml

```diff
@@ -74,15 +69,91 @@
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-tools package
+        shell: apt depends scylla-tools={{ scylla_version }}*
+        register: apttoolsdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllatoolsdependencies: "{{ apttoolsdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-server package
+        shell: apt depends scylla-server={{ scylla_version }}*
+        register: aptserverdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllaserverdependencies: "{{ aptserverdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Install specified OSS Scylla packages
+        apt:
+          name: "{{ item }}={{ scylla_version }}*"
+          state: present
+        with_items: "{{ scyllaserverdependencies + scyllatoolsdependencies + scylladependencies }}"
+
+      - name: Install meta Scylla package
+        apt:
+          name: "scylla={{ scylla_version }}*"
+          state: present
+
     when: scylla_version != 'latest' and scylla_edition == 'oss'
 
+#apt list -a scylla-enterprise | grep 2021.1.9
+#WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
+#scylla-enterprise/stable 2021.1.9-0.20220208.86e3ea4df-1 amd64
+
+
   - name: Install specified Enterprise Scylla
-    apt:
-      name: "scylla-enterprise={{ scylla_version }}*"
-      state: present
+    block:
+      - name: Get dependencies of scylla-enterprise package
+        shell: apt depends scylla-enterprise={{ scylla_version }}*
+        register: aptdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scylladependencies: "{{ aptdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-enterprise-server package
+        shell: apt depends scylla-enterprise-server={{ scylla_version }}*
+        register: aptserverdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllaserverdependencies: "{{ aptserverdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-enterprise-tools package
+        shell: apt depends scylla-enterprise-tools={{ scylla_version }}*
+        register: apttoolsdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllatoolsdependencies: "{{ apttoolsdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Install specified Enterprise Scylla packages
+        apt:
+          name: "{{ item }}={{ scylla_version }}*"
```

Comment:
Here it's the same: versions are per package and they may be different for each separate package. Learn them from `apt details`